pkgs/top-level/python-packages.nix

```diff
@@ -6157,6 +6157,8 @@ in {
 
   stringcase = callPackage ../development/python-modules/stringcase { };
 
+  vcard = callPackage ../tools/text/vcard { };
```

Comment:
I would move the expression under python-modules, and then just add
```
disabled = isPy27;
```
to show that it's only python3 supported.

This would also mean removing python3Packages and importing the dependencies you need directly by name.

Reply:
Thank you for reviewing! After adding `disabled = isPy27` and changing `with python3Packages` to `with pythonPackages` (because otherwise it's specified twice that it's a Python 3 package) I get this error:

> error: vcard-0.13.0 not supported for interpreter python2.7

Is this intentional? Do I need to specify twice that it's Python 3 only?

Reply:
I believe that error is because when you use `pythonPackages`, `toPythonApplication` tries to check that it can run your application with python 2, sees that this is disabled, then runs an error.